src/main/java/org/ml_methods_group/plugin/AutomaticRefactoringAction.java

```diff
@@ -166,12 +168,20 @@ private void updateResults(@NotNull RefactoringExecutionContext context) {
     private void showDialogs(@NotNull RefactoringExecutionContext context) {
         updateResults(context);
         final Set<String> selectedAlgorithms = ArchitectureReloadedConfig.getInstance().getSelectedAlgorithms();
-        final List<AlgorithmResult> algorithmResult = context.getAlgorithmResults()
+        final List<AlgorithmResult> algorithmResults = context.getAlgorithmResults()
                 .stream()
                 .filter(result -> selectedAlgorithms.contains(result.getAlgorithmName()))
                 .collect(Collectors.toList());
-        ServiceManager.getService(context.getProject(), RefactoringsToolWindow.class)
-                .show(algorithmResult, context.getEntitySearchResult(), context.getScope());
+        boolean isAllEmpty = true;
```

Comment:
`boolean notEmptyResult = algorithmResults.stream().flatMap(...).findAny().isPresent();`

and invert the condition below